homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```

Comment:
Please change the name of the file back to `index.html` - it is the main starting point for any web-site

Reply:
And, BTW, all these files should be placed in GamesProject, not in this
<img width="417" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/1268cfc4-7bcb-4204-97c5-206a1c8dce54">

Reply:
fixed